tutorials/3d/global_illumination/using_lightmap_gi.rst

```diff
@@ -645,6 +645,14 @@ Reducing LightmapGI artifacts
 
 If you notice LightmapGI nodes popping in and out of existence as the camera
 moves, this is most likely because the engine is rendering too many LightmapGI
-instances at once. Godot is limited to rendering 8 LightmapGI nodes at once,
-which means up to 8 instances can be in the camera view before some of them will
-start flickering.
+instances at once.
+
+Here are the limits for each renderer:
+
+- **Forward+**: Up to 8 LightmapGI nodes can be rendered at once.
+- **Forward Mobile**: Up to 2 LightmapGI nodes can be rendered at once.
+- **Compatibility**: An unlimited number of LightmapGI nodes can be rendered at once.
```

Comment:
```suggestion
- **Compatibility**: No hard limit currently however that may change. Design your scene as if 8 is the maximum
```

@clayjohn would something like this be ok?

Reply:
No, there is a hard limit, someone just needs to go look at the code to figure out what it is. 

Not knowing the limit is not the same as having an infinite limit.